h/services/group_create.py

```diff
@@ -105,7 +105,7 @@ def _create(self, name, userid, type_flags, scopes, **kwargs):
 
         group_scopes = [GroupScope(scope=s) for s in scopes]
 
-        if "organization" in kwargs:
+        if kwargs.get("organization"):
```

Comment:
Allow `GroupCreateService.create_*_group()` to be called with an `organization=None` kwarg and this will have the same effect as if it were called with no `organization` kwarg at all. This is convenient from the caller's point of view (the caller is the admin pages view/schema code).